### ISSUE TITLE
Problem: /currency/:slug is catching errors (ethereum)

### ISSUE THREAD

**Issue by janat08** [Blockrazor/blockrazor#471]
Problem: There're errors cought, in which file i don't know. Likely caused by #465
```
Exception in queued task: Error: Expected template or null, found: df hfdg hfdh fdgh 
    at Blaze.View._render (http://localhost:3000/packages/spacebars.js?hash=ebf9381e7fc625d41acb0df14995b7614360858a:61:13)
    at Blaze.View.doRender (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:2086:25)
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:1934:20
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:3744:12)
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:1932:29
    at Object.Blaze._withCurrentView (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:2271:12)
    at viewAutorun (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:1931:18)
    at Tracker.Computation._compute (http://localhost:3000/packages/tracker.js?hash=997515fa2d5b0530ba07741da556c4b36963ef3b:339:36)
    at new Tracker.Computation (http://localhost:3000/packages/tracker.js?hash=997515fa2d5b0530ba07741da556c4b36963ef3b:229:10)
    at Object.Tracker.autorun (http://localhost:3000/packages/tracker.js?hash=997515fa2d5b0530ba07741da556c4b36963ef3b:613:11)
```

**Comment by gazhayes** [comment]
ping @anbud

**Comment by anbud** [comment]
On it.

**Comment by anbud** [comment]
Done in #498.

**Comment by gazhayes** [comment]
@janat08 can you test to make sure this is solved, then close it.

**Comment by janat08** [comment]
"Exception in template helper: TypeError: val[0].join is not a function" fixed in #509
